### ISSUE TITLE
PropertyStoreSpringJdbc not returning property on UI

### ISSUE THREAD

**Issue by cboulme** [ff4j/ff4j#509]
When using PropertyStoreSpringJdbc as feature store for ff4j, the custom properties are not returned by the readAll method. 
And so they are not displayed on the UI.
This is working fine when using JdbcFeatureStore instead:
https://github.com/ff4j/ff4j/blob/master/ff4j-store-springjdbc/src/main/java/org/ff4j/springjdbc/store/FeatureStoreSpringJdbc.java#L365 

https://github.com/ff4j/ff4j/blob/master/ff4j-core/src/main/java/org/ff4j/store/JdbcFeatureStore.java#L387

**Comment by clun** [comment]
You are correct the SpringJdbc store needs some extra queries to populate the features.

```java
for (Feature flipPoint : lFp) {
      // To be added
       readProperties(flipPoint);
    // <--
       mapFP.put(flipPoint.getUid(), flipPoint);
}
```
